### PR TITLE
update holi and diwali holidays till 2030

### PR DESCRIPTION
**Pull request by raajtilaksarma**
Updated holi and diwali holidays till the year 2030. Added a source(link) for Holi dates till 2030.

### CODE REVIEW

File: python/fbprophet/hdays.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def _populate(self, year):
         # http://www.theholidayspot.com/diwali/calendar.htm
         # https://www.timeanddate.com/holidays/india/diwali?starty=
         # https://www.infoplease.com/calendar-holidays/major-holidays/
+        # https://www.learnreligions.com/when-is-holi-1770208
         warning_msg = "We only support Diwali and Holi holidays from 2010 to 2025"
```
Comment (joseangel-sc) [inline python/fbprophet/hdays.py]: I guess this needs an update then? ;)

Comment (raajtilaksarma) [reply python/fbprophet/hdays.py]: Yes. Adding a commit.

Comment (raajtilaksarma) [reply python/fbprophet/hdays.py]: Changed it.

**Comment by bletham** [comment]
Great, thanks!

**Pull request merged**